### ISSUE TITLE
Advanced preferences to control referrer options

### ISSUE THREAD

**Issue by wilkowy** [RequestPolicyContinued/requestpolicy#796]
The referrer options are also important for privacy and as RP's preferences UI offers control over:

- network.dns.disablePrefetch
- network.prefetch-next
- network.http.speculative-parallel-limit

than maybe it could also offer checkboxes/dropdowns for:

- network.http.referer.XOriginPolicy
- network.http.referer.spoofSource
- network.http.referer.trimmingPolicy
- network.http.sendRefererHeader
- network.http.sendSecureXSiteReferrer

**Comment by myrdd** [comment]
I agree, RP could have more _recommended prefs_. Still, I'd like to improve the system of "recommended prefs" (#749) before cluttering the settings page with checkboxes.

**Comment by dgplayer** [comment]
This functionality is already available in the combination of [CanvasBlocker, Random Agent Spoofer and RefControl].
Still, it would be nice to have the level of control over spoofing that RequestPolicy Continued provides.
